google-cloud-secretmanager-v1beta1/google-cloud-secretmanager-v1beta1.gemspec

```diff
@@ -25,6 +25,7 @@ Gem::Specification.new do |gem|
 
   gem.add_dependency "gapic-common", "~> 0.0"
   gem.add_dependency "google-cloud-core", "~> 1.1"
+  gem.add_dependency "grpc-google-iam-v1", "~> 0.6.9"
```

Comment:
Should this dependency be loosened so that a future 0.7.0 version can be used?

Reply:
Not for now. We don't have reason to update it yet. (I don't know of an impending 0.7.0 version, nor what it would include or what its compatibility would be.)